app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java

```diff
@@ -57,4 +57,26 @@ public String toString() {
     public boolean isCoveringLens() {
         return (redLevel <= MIN_RED_LEVEL) || (confidence >= 0.5);
     }
+
+    public boolean isPressureExcessive() {
+        int hr = bpm;
+
+        if(hr > 250 || hr <= 40) {
+            return true;
+        }
+
+        // for testing purposes
+        return true;
+        //return false;
```

Comment:
Should this be false by default?

Reply:
Yes, but for testing purposes I kept it as true